var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -53,6 +53,14 @@ class PyScipy(PythonPackage):
     version("1.2.1", sha256="e085d1babcb419bbe58e2e805ac61924dac4ca45a07c9fa081144739e500aa3c")
     version("1.1.0", sha256="878352408424dffaa695ffedf2f9f92844e116686923ed9aa8626fc30d32cfd1")
 
+    # Based on wheel availability on PyPI
+    depends_on("python@3.8:3.11", when="@1.9.2:", type=("build", "link", "run"))
+    depends_on("python@3.8:3.10", when="@1.8:1.9.1", type=("build", "link", "run"))
+    depends_on("python@:3.10", when="@1.7.2:1.7", type=("build", "link", "run"))
+    depends_on("python@:3.9", when="@1.5.4:1.7.1", type=("build", "link", "run"))
+    depends_on("python@:3.8", when="@1.3.2:1.5.3", type=("build", "link", "run"))
+    depends_on("python@:3.7", when="@1.1:1.3.1", type=("build", "link", "run"))
```

Comment:
Is dropping the lower bounds here done for simplicity? Or are Python versions up to Python 3.7 no longer available so this is safe?

Reply:
Both. It gets more complicated when you have to split up `2.7,3.4:3.7`, `3.4:3.7`, `3.5:3.7`, etc. We only support 3.7+, so it doesn't make a difference anyway.

Reply:
Great. LGTM then.